stacks/SiriGeneratorStack.ts

```diff
@@ -11,10 +11,13 @@ export function SiriGeneratorStack({ stack }: StackContext) {
         name: `cdd-siri-sx-${stack.stage}`,
     });
 
+    const siriSXUnvalidatedBucket = new Bucket(stack, "SiriSXUnvalidatedBucket", {
+        name: `cdd-siri-sx-unvalidated-${stack.stage}`,
```

Comment:
It's worth enabling encryption and block public access on both of these buckets:
```
cdk: {
    bucket: {
        encryption: BucketEncryption.S3_MANAGED,
        blockPublicAccess: {
            blockPublicAcls: true,
            blockPublicPolicy: true,
            ignorePublicAcls: true,
            restrictPublicBuckets: true,
        },
    },
},
```

Reply:
done